js/util.ts

```diff
@@ -125,3 +125,23 @@ export function deferred(): Deferred {
     reject: reject!
   };
 }
+
+// tslint:disable-next-line:no-any
```

Comment:
Would suggest the following:

```ts
const TypedArray = Object.getPrototypeOf(Uint8Array);
export function isTypedArray(x: unknown): x is TypedArray {
  return x instanceof TypedArray;
}
```

Reply:
That works, but wouldn't that mess up the TS compiler idea of what a TypedArray is? Currently it's defined as a union of all the various others.

https://github.com/denoland/deno/blob/b2bc0a7fc9291d1131c9b577768608e7911ea931/js/types.ts#L2

I expect using the above `TypedArray` would give TS errors when doing a function like this
```ts
function foo(ta: TypedArray) { ; }
foo(new Int32Array())
```